Resources/template.json

```diff
@@ -40,6 +40,9 @@
         },
         "templateBaseUrl": {
             "type": "string"
+        },
+        "azureMapSubscriptionKey": {
+            "type": "string"
```

Comment:
Should this be a string or securestring?

Reply:
changed to securestring